app/cdap/components/PostRunActions/PostRunActionsWizard.js

```diff
@@ -99,7 +146,105 @@ export default class PostRunActionsWizard extends Component {
   };
 
   render() {
-    let action = this.props.action;
+    let action = this.state.action;
+    let currentStage = 1;
+    if (!this.props.isDeployed) {
+      return (
+        <Modal
+          isOpen={this.props.isOpen}
+          toggle={this.toggleAndPreventPropagation}
+          backdrop="static"
+          modalClassName="post-run-actions-modal hydrator-modal node-config-modal"
+          zIndex={MATERIAL_UI_Z_INDEX + 1} // Workaround for material-ui PipelineModeless
+        >
+          <div className="modal-header clearfix">
+            {action ? (
+              <h5 className="modal-title pull-left">
+                <span>{action.plugin.name || action.name}</span>
+                <small className="plugin-version">
+                  {action.version || action.plugin.artifact.version}
+                </small>
+                <p>
+                  <small>{action.description}</small>
+                </p>
+              </h5>
+            ) : (
+              <h4 className="modal-title pull-left">Alerts</h4>
+            )}
+            {this.state.errorCount === 0 && <span className="text-success">No errors found.</span>}
+            {this.state.errorCount === 1 && <span className="text-danger">1 error found</span>}
+            {this.state.errorCount > 1 && (
+              <span className="text-danger">{this.state.errorCount} errors found</span>
+            )}
+            <div className="btn-group pull-right">
+              {this.state.showValidateButton && (
+                <button
+                  className="btn btn-primary validate-btn"
+                  data-cy="validate-btn"
+                  type="button"
+                  onClick={this.validatePluginProperties.bind(this)}
```

Comment:
instead of binding this, you should be able to change the definition of validatPluginProperties to 

validatePluginProperties = () => etc